.golangci.yml

```diff
@@ -182,22 +156,16 @@ linters:
     - gochecknoinits
     - gocritic
     - gofmt
-    - golint
     - gosec # Should turn back on soon
-    - interfacer
     - lll
-    - maligned
     - prealloc # Should turn back on soon
-    - scopelint
```

Comment:
Do these linters not exist anymore or?

Reply:
They do not, nope. Also CLA should be signed.

Reply:
FYI: CI is broken atm, see [this](https://github.com/element-hq/dendrite/actions/runs/14821949158/job/41610128777?pr=3569#step:5:27)